.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,10 @@
+---
+name: Bug report
+about: To report a part of mocha not working as expected
+title: ''
+labels: 'unconfirmed-bug'
+---
+
 <!--
 Have you read Mocha's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/mochajs/mocha/blob/master/.github/CODE_OF_CONDUCT.md
 For more, check out the Mocha Gitter chat room: https://gitter.im/mochajs/mocha
```

Comment:
| **Actual behavior:** [What actually happens]

**Actual behavior:** [What actually happens]
<!--
Please include any output, especially error messages (including stacktrace). Remember, we can't see your screen.
Scrub if needed so as not to reveal passwords, etc.
-->